src/code/z_room.c

```diff
@@ -1,94 +1,578 @@
 #include "global.h"
 
-void Room_nop8012D510(PlayState* play, Room* room, UNK_PTR param_3, UNK_TYPE1 param_4) {
+void Room_Update(PlayState* play, Room* room, Input* input, s32 arg3) {
 }
 
-void Room_DrawType3Mesh(PlayState* play, Room* room, u32 flags) {
+void Room_DrawNone(PlayState* play, Room* room, u32 flags) {
 }
```

Comment:
I assume these are based on usage?

Reply:
Yeah, they're both inferred. `Room_Update` because it's called from `Play_Update`, there's no other update function, and presumably you'd want to put anything you want to update in this function. Just an assumption though, The arg for `Input` is weird as well. Could always revert to just a `Noop` if we want.

`Room_DrawNone` should at least follow the functions in the draw handlers, so could also do `Room_DrawNoop` here

Reply:
the fact that it takes a controller input arg lends itself to being something related to debugging that was ifdefed out. imo not enough info to say

Reply:
I would say `Room_HandleLoadCallbacks` is the closet thing to a room update (and ive already voiced my concerns about this name in the past, theres no callbacks really happening here)

Reply:
That's fair about the debug interpretation of `Input`, I can change to just `Room_Noop`, it doesn't really change all that much since it's just speculation.

I'm also definitely not attached to `Room_HandleLoadCallbacks`, but I was going to wait to change this name later in hopes of an OoT discussion to see what kind of names come up, particularly by those who work with Fast64 and really understand scenes/rooms quite well. However, if there's a good name we can come up with now, we can change it.

Reply:
it can for sure wait for oot. just mentioning that function (which is also called by Play_Update) is moreso the update for rooms (even though in this context, updating a room just means doing a few things on new room load)